app/backend/src/couchers/query.py

```diff
@@ -0,0 +1,59 @@
+from sqlalchemy import or_
+from sqlalchemy.orm import Query, aliased
+
+from couchers.models import User, UserBlock
+from couchers.utils import is_valid_email, is_valid_user_id, is_valid_username
+
+
+class CouchersQuery(Query):
+    def _blocked_users(self, session, user_id):
```

Comment:
move to top level, no need to have it as a member function. It doesn't chain another filter onto the query.